apps/ex_nvr_web/lib/ex_nvr_web/live/user_list_live.ex

```diff
@@ -0,0 +1,143 @@
+          >
+            <svg
+              class="w-5 h-5"
+              aria-hidden="true"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M10 6a2 2 0 110-4 2 2 0 010 4zM10 12a2 2 0 110-4 2 2 0 010 4zM10 18a2 2 0 110-4 2 2 0 010 4z" />
+            </svg>
+            <div
+              id={"dropdownDots_#{user.id}"}
+              class="z-10 hidden text-left bg-white divide-y divide-gray-100 rounded-lg shadow w-44 dark:bg-gray-700 dark:divide-gray-600"
+            >
+              <ul
+                class="py-2 text-sm text-gray-700 dark:text-gray-200"
+                aria-labelledby={"dropdownMenuIconButton_#{user.id}"}
+              >
+                <li>
+                  <.link
+                    href={~p"/users/#{user.id}"}
+                    class="block px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-white"
+                  >
+                    Update
+                  </.link>
+                </li>
+                <li>
+                  <.link
+                    href="#"
+                    class="block px-4 py-2 hover:bg-gray-100 dark:hover:bg-gray-600 dark:hover:text-red"
+                    phx-click="confirm-delete"
+                    phx-value-id={user.id}
+                    phx-value-confirm="Are you sure you want to delete this user?"
+                  >
+                    Delete
+                  </.link>
+                </li>
+              </ul>
+            </div>
+          </.button>
+          <%= if @confirm_delete && @user_id == user.id do %>
+            <div class="fixed top-0 right-0 bottom-0 left-0 flex justify-center items-center bg-black bg-opacity-70 z-50">
+              <div class="confirm-dialog bg-white p-4 rounded-lg text-center border-b border-gray-200 dark:bg-gray-800 dark:border-gray-700">
+                <p class="text-white mb-10 mt-10">
+                  <%= "Are you sure you want to delete the User: '#{user.email}'" %>
+                </p>
+                <.button
+                  phx-click="delete"
+                  phx-value-id={@user_id}
+                  class="bg-red-500 text-white px-4 py-2 mx-2 rounded hover:bg-red-600"
+                >
+                  Yes
+                </.button>
+                <.button
+                  phx-click="cancel-delete"
+                  class="bg-gray-500 text-white px-4 py-2 mx-2 rounded hover:bg-gray-600"
+                >
+                  No
+                </.button>
+              </div>
+            </div>
+          <% end %>
+        </:action>
+      </.table>
+    </div>
+    """
+  end
+
+  def mount(_params, _session, socket) do
+    {:ok, assign(socket, users: Accounts.list(), confirm_delete: false, user_id: nil)}
+  end
+
+  def handle_event("confirm-delete", %{"id" => id}, socket) do
+    {:noreply, socket |> assign(confirm_delete: true, user_id: id)}
```

Comment:
I don't think we need this actions. We should show the modal on front-end, it it clicks `ok` delete the user, otherwise just hide the modal.